rclcpp/CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ set(${PROJECT_NAME}_SRCS
   src/rclcpp/network_flow_endpoint.cpp
   src/rclcpp/node.cpp
   src/rclcpp/node_interfaces/node_base.cpp
+  src/rclcpp/node_interfaces/node_builtin_executor.cpp
```

Comment:
Please consider if this file is necessary to locate in `node_interfaces` directory since `node_builtin_executor` is not exported as a node interface.